doc/source/contributing/contribution_guidelines.rst

```diff
@@ -95,6 +54,53 @@ Licensing
+
+
+Requesting a Feature
+--------------------
+Find the *Issues* tab at the top of GitHub repository and click *New Issue* button. Please give your suggestion a clear title and let us know if this is something you'd like to work on and contribute.
+
+
+Reporting a bug
+---------------
+Find the *Issues* tab at the top of GitHub repository and click *New Issue* button. Give your issue a clear title and describe the steps required to recreate it in as much detail as possible. If you can, include a small example that reproduces the error. More information and minimal examples will help us resolve issues faster.
+
+
+Questions and Help
+------------------
+
+Please do not create issues to ask for help. A faster way to reach the community is through our Science/ICESat-2 subcategory on the `Pangeo discourse <https://discourse.pangeo.io/c/science/icesat-2/16>`_ page. We are excited to have you join an existing conversation or start a new post! Please note that a GitHub login is required to post on the discourse page.
+
+Other Resources
+^^^^^^^^^^^^^^^
+- Check out our :ref:`ICESat-2 Open-Source Resources Guide<resource_ref_label>` for a host of tools and code for getting and working with ICESat-2 data
+- The ICESat-2 Hackweek Tutorial repositories are great resources for learning basic Python and development skills, learning about ICESat-2 data, setting up a computational environment, and finding and analyzing ICESat-2 datasets.
+    - 2019 repo: <https://github.com/ICESAT-2HackWeek/ICESat2_hackweek_tutorials>`_
+    - 2020 repo: <https://github.com/ICESAT-2HackWeek/2020_ICESat-2_Hackweek_Tutorials>`_
+    - 2022 repo: <https://github.com/ICESAT-2HackWeek/website2022>`_
+- A great set of `interactive tutorials <https://learngitbranching.js.org>`_ for learning and practicing using git
+- Let us know about the helpful tools you've found by posting on our discourse forum as indicated above (`Questions and Help`_)!
+
+
+Adding Examples
+---------------
+We are delighted you'd like to contribute your `icepyx <https://github.com/icesat2py/icepyx>`_ example! Examples may be in the form of executable scripts or interactive Jupyter Notebooks. Please make sure that each example has a descriptive name so someone not familiar with your project understands its general behavior. Fully working examples should be submitted using a pull request to the "development" branch, following the steps outlined above for `Contributing Code`_.
+
+
+Add ancillary dataset
+---------------------
+If you have an idea for a dataset that complements ICESat-2 data, please consider adding search and download capabilities through our QUEST module, which is written in object oriented code and designed to allow easy incorporation of other datasets. Stay tuned for a tutorial on how to add your dataset.
```

Comment:
Grammar alert (the internet doesn't seem to have a clear opinion on dashed or not). @RomiP?
```suggestion
If you have an idea for a dataset that complements ICESat-2 data, please consider adding search and download capabilities through our QUEST module, which is written in object-oriented code and designed to allow easy incorporation of other datasets. Stay tuned for a tutorial on how to add your dataset.
```

Reply:
note we will use the hyphen!